ros_gz_bridge/test/utils/gz_test_msg.cpp

```diff
@@ -76,6 +76,116 @@ void compareTestMsg(const std::shared_ptr<gz::msgs::Color> & _msg)
   EXPECT_EQ(expected_msg.a(), _msg->a());
 }
 
+void createTestMsg(gz::msgs::DVLBeamState & _msg)
+{
+  _msg.set_id(1.0);
+  createTestMsg(*_msg.mutable_velocity());
+  createTestMsg(*_msg.mutable_range());
+  _msg.set_rssi(100.0);
+  _msg.set_nsd(200.0);
+  _msg.set_locked(true);
+}
+
+void compareTestMsg(const std::shared_ptr<gz::msgs::DVLBeamState> & _msg)
+{
+  gz::msgs::DVLBeamState expected_msg;
```

Comment:
include msg header

Reply:
Added in [89b14aa](https://github.com/gazebosim/ros_gz/pull/789/commits/89b14aa10731aed70f7ae7acdbe1c5ce8698f6f4).